docs/table/performance.inc.rst

```diff
@@ -39,3 +39,23 @@ then use ``serialize_method='data_mask'``.
 It uses the non-masked version of data and it is faster::
 
     >>> tm.write('tm.ecsv', overwrite=True, serialize_method='data_mask') # doctest: +SKIP
+
+Reading a ``.fits`` in |Table| using
+:meth:`~astropy.table.Table.read` when ``memmap=False`` can be very slow
+because it reads full |Table| in the memory ::
+
+    >>> import numpy as np
+    >>> from astropy.table import Table
+    >>> tbl = Table({
+    ...     'a': np.arange(1e7),
+    ...     'b': np.arange(1e7, dtype=float),
+    ...     'c': np.arange(1e7, dtype=float),
+    ...  })
+    >>> tbl.write(tmpfile, overwrite=True) # doctest: +SKIP
+    >>> t = Table.read('test.fits', memmap=False) # doctest: +SKIP
```

Comment:
I don't think we need to show to slow way. So I would suggest to merge the two paragraph into one, as well as have only one example.